src/Compute/Compute.Test/ScenarioTests/VirtualMachineTests.cs

```diff
@@ -234,7 +234,14 @@ public void VirtualMachineGetStatusWithHealhtExtension()
         {
             TestRunner.RunTestScript("Test-VirtualMachineGetStatusWithHealhtExtension");
```

Comment:
Each ADO bug or work item should have a PR task that has a link to the PR, please add that in all of your bugs.

Reply:
I tried but this ADO somehow does not have link to PR. Could you please confirm this? there's ADO link in the description